backend/cms/models/configuration.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+
+class Configuration(models.Model):
+    key = models.CharField(max_length=100, unique=True, blank=False)
+    value = models.CharField(max_length=1000, unique=True, blank=False)
```

Comment:
Is the unique flag really required for the value?

Reply:
Oops.  :) Will fix.